src/common/rbac/project/util.go

```diff
@@ -47,7 +115,22 @@ func policiesForPublicProject(namespace rbac.Namespace) []*rbac.Policy {
 func policiesForSystemAdmin(namespace rbac.Namespace) []*rbac.Policy {
```

Comment:
What is the different with `GetAllPolicies ()`?

Reply:
`policiesForSystemAdmin` removed